src/devices/MAX31856/samples/MAX31856.Sample.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using System.Threading;
+using Iot.Device.MAX31856;
+
+SpiConnectionSettings settings = new(0, 0)
+{
+    ClockFrequency = MAX31856.SpiClockFrequency,
+    Mode = MAX31856.SpiMode,
+    DataFlow = 0
+};
+
+using SpiDevice device = SpiDevice.Create(settings);
+using MAX31856 sensor = new(device, ThermocoupleType.K);
+while (true)
+{
+    // read temperature
+    var temp = sensor.GetTemperature().DegreesFahrenheit;
```

Comment:
As pointed in the Readme, catch the exception or better, use a TryGetTemperature pattern